README.md

```diff
@@ -143,3 +143,9 @@ If you like this project, please give it a star ⭐️
 I would like to thank <a href="https://www.microverse.org/">Microverse</a> for providing the guidelines that made this project possible, <a href="https://www.behance.net/sakwadesignstudio">Nelson Sakawa</a> for providing the <a href="https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)">design</a> for the user interface, and the folks at <a href="https://coincap.io/">CoinCap</a> for making the <a href="https://docs.coincap.io/">API</a> they use freely available to the public.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]() licensed.
```

Comment:
- Hey, great job adding the MIT license, however, you've not added the link so it's clickable and users can navigate to read it from the readme file. e.g [MIT](the link here). please, fix this to have your project approved.